### PR TITLE
Bonus gmx UI fixes

### PR DESCRIPTION
**Pull request by onemicky**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206349481659089

### CODE REVIEW

File: src/components/ModalInfo/ModalInfo.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+    img {
+      display: block;
+    }
+  }
+
+  &-text {
+    font-size: 1.3rem;
+  }
+}
```
Comment (vipineth) [inline src/components/ModalInfo/ModalInfo.scss:21]: On branch `release-13` there is a component named `AlertInfo` which is coming form the SLTP modal PR. I think `ModalInfo` component can be replaced by the `AlertInfo` component if that's okay.

File: src/lib/numbers.ts [hunk 2]
```diff
@@ -7,6 +7,8 @@ import { getPlusOrMinusSymbol } from "./utils";
 const MAX_EXCEEDING_THRESHOLD = "1000000000";
 const MIN_EXCEEDING_THRESHOLD = "0.01";
 
+export const BIG_ZERO = BigNumber.from(0);
```
Comment (vipineth) [inline src/lib/numbers.ts:10]: I think a better name would be `BN_ZERO` or you can use `ethers.constants.Zero`

**Review by vipineth — CHANGES_REQUESTED** [review]

File: src/lib/numbers.ts (hunk 2 above)
Comment (onemicky) [reply src/lib/numbers.ts]: Good thought! Switched to BN_ZERO as the idea was to have a short constant name

File: src/components/ModalInfo/ModalInfo.scss (hunk 1 above)
Comment (onemicky) [reply src/components/ModalInfo/ModalInfo.scss:21]: Hm, there really is a good component with a same functionality. By the way this task is going to be merge directly to master and goes off-release so I don't know a good way to take this component from release-13 don't you?
May we merge this PR as is and i will replace ModalInfo separately after release-13 merging?

**Review by vipineth — APPROVED** [review]

File: src/lib/numbers.ts (hunk 2 above)
Comment (gmxer) [reply src/lib/numbers.ts]: fyi, we use ZERO in subgraph

image

maybe we can do similar in UI, I'm not sure really though

**Pull request merged**